### PR TITLE
Add `README.md` to watchlist

### PR DESCRIPTION
**Pull request by jakirkham**
xref: https://github.com/dask/dask-docker/pull/194#issuecomment-962985426

### CODE REVIEW

File: .github/workflows/watch-conda-forge.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Find and Replace Release
         uses: jacobtomlinson/gha-find-replace@0.1.1
         with:
-          include: '\.github\/workflows\/build\.yml|build\/docker-compose\.yml'
+          include: 'README\.md|\.github\/workflows\/build\.yml|build\/docker-compose\.yml'
```
Comment (jakirkham) [inline .github/workflows/watch-conda-forge.yml:25]: From running locally

```bash
$ find . -not -path "./.git/*" -type f | grep -iE 'README\.md|\.github\/workflows\/build\.yml|build\/docker-compose\.yml'
./README.md
./.github/workflows/build.yml
./build/docker-compose.yml
```

**Comment by jakirkham** [comment]
cc @jacobtomlinson

**Review by jacobtomlinson — APPROVED** [review]

**Comment by jacobtomlinson** [comment]
Thanks @jakirkham

**Comment by jrbourbeau** [comment]
Thanks @jakirkham!

**Pull request merged**